docs/requirements.txt

```diff
@@ -1,2 +1,3 @@
 m2r
 sphinxcontrib-apidoc
+docutils<=0.17.1
```

Comment:
Hmm, it seems that actually this hunk fixes the RTD build, not only my local test with tox.